tox.ini

```diff
@@ -0,0 +1,8 @@
+[pycodestyle]
+ignore = E111,E114,E731
```

Comment:
Consider adding some text detailing what these codes mean, or alternatively, add this link:
https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes

Reply:
Done.